static/src/js/components/ChunkedOrderModal/ChunkedOrderModal.js

```diff
@@ -70,6 +70,7 @@ export class ChunkedOrderModal extends Component {
       </PortalLinkContainer>
     )
 
+    // TODO this message needs to represent either defaultGranulesPerOrder or maxItemsPerOrder for echo orders
```

Comment:
Did you mean to keep this? If so, we should add a ticket to track the improvement.